<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tenduke::qt::demo::AutoDiscoveredParameters</class>
 <widget class="QMainWindow" name="tenduke::qt::demo::AutoDiscoveredParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpAutoDiscovery">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Auto-discovery</string>
    </property>
    <widget class="QLabel" name="lblHost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Host</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDiscover">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>220</y>
       <width>85</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Discover</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAuthorizationEndpoint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Authorization endpoint</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblAuthorizationEndpoint_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Token endpoint</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblAuthorizationEndpoint_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>135</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Userinfo endpoint</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblAuthorizationEndpoint_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Issuer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="hostname">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>https://genco.10duke.com</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="authorizationEndpoint">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>65</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tokenEndpoint">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="userinfoEndpoint">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>135</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="issuer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpOAuthClientConfiguration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>780</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>OAuth client configuration</string>
    </property>
    <widget class="QLabel" name="lblClientId">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Client id</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblClientSecret">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Client secret</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblRedirectURI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Redirect URI</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="clientIdentifier">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>QTDemo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="clientSecret">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>65</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>FQdL!XSYYrQ7-zeiCe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="redirectUri">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>tenduke://callback</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>429</y>
      <width>781</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Licensing configuration</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Licensing endpoint</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblHardwareId">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Hardware id</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="licensingEndpoint">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>https://genco.10duke.com/authz</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hardwareId">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>65</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>simulated-hardware-id</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnGo">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
